starter-code/src/app/services/access-control-log.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AccessControlLogService {
+
+  // Array<Object> but we use 'any' to avoid eslint correction
+  logMessages: any = [{
```

Comment:
You don't specify the types of object keys. Set the type with Array<any> (or create a model, but forget about that for now).